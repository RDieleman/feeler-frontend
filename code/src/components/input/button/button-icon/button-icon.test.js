import React from "react";
import {shallow} from "enzyme";
import {ButtonIconComponent} from "./button-icon.component";

it("Expect to render ButtonIcon component", () => {
    expect(shallow(
        <ButtonIconComponent
            handleOnClick={() => "Button icon clicked"}
            iconUrl={""}
        />)).toMatchSnapshot();
});