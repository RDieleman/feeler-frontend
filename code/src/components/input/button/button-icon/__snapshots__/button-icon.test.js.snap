// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render ButtonIcon component 1`] = `ShallowWrapper {}`;
